src/components/clusterprovider/common/form.ts

```diff
@@ -1,11 +1,17 @@
 import { formStyles, styles, waitScript } from '../../../wizard';
+import { NEXT_FN } from '../../wizard/wizard';
+import { SENDING_STEP_KEY } from '../clusterproviderserver';
 
 // HTML rendering boilerplate
 
+const DO_NOT_PROPAGATE = ['nextStep', SENDING_STEP_KEY];
+
 export function propagationFields(previousData: any): string {
     let formFields = "";
     for (const k in previousData) {
-        formFields = formFields + `<input type='hidden' name='${k}' value='${previousData[k]}' />\n`;
+        if (DO_NOT_PROPAGATE.indexOf(k) < 0) {
```

Comment:
why not `DO_NOT_PROPOGATE.includes(k)` ?

Reply:
TypeScript gives me an error if I use `includes`.  I'm not sure why that is.  Perhaps I need to update the target JS version in the TypeScript config, but I'm going to call 'out of scope' on that for this PR...